--- tmux-1.3-orig/GNUmakefile	2010-07-19 07:54:24.320563707 +0200
+++ tmux-1.3/GNUmakefile	2010-07-19 07:58:25.390541271 +0200
@@ -81,7 +81,7 @@
 install:	all
 		$(INSTALLDIR) $(DESTDIR)$(PREFIX)/bin
 		$(INSTALLBIN) tmux $(DESTDIR)$(PREFIX)/bin/tmux
-		$(INSTALLDIR) $(DESTDIR)$(PREFIX)/man/man1
-		$(INSTALLMAN) tmux.1 $(DESTDIR)$(PREFIX)/man/man1/tmux.1
+		$(INSTALLDIR) $(DESTDIR)$(PREFIX)/share/man/man1
+		$(INSTALLMAN) tmux.1 $(DESTDIR)$(PREFIX)/share/man/man1/tmux.1
 
 -include .depend
